.github/workflows/templates-build-push-image.yml

```diff
@@ -1,5 +1,6 @@
+name: Build and Push (Linux)
```

Comment:
Let's revert this. Also, you can just use the public image that is already there (as mentioned :))